frontend/public/components/monitoring.tsx

```diff
@@ -418,7 +422,7 @@ const SilenceTableRow: RowFunction<Silence> = ({ index, key, obj, style }) => {
         </div>
       </TableData>
       <TableData className={tableSilenceClasses[1]}>
-        {_.isEmpty(firingAlerts) ? '-' : <SeverityCounts alerts={firingAlerts} />}
+        <SeverityCounts alerts={firingAlerts} />
```

Comment:
`-` was already shown on the silence list page, so moved that into `SeverityCounts` so that it also applies to the details page.